### DISCUSSION TITLE
Is there an alternative to get the current_app?

### DISCUSSION THREAD

**Discussion by selobu** [fastapi/fastapi#6314] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [x] I commit to help with one of those options 👆

### Example Code

```python
# Example
from flask import current_app

# required
from fastapi import current_app
```

### Description

I need to get the current_app instantiated by using app=FasAPI() in diferent functions and events so I would like to know if there is an alternative similar to the exitent one in flask

### Operating System

Linux

### FastAPI Version

latest

### Python Version

3.8

**Comment by JarroVGIT** [comment, marked as answer]
You are looking for the app object, which is part of the `Request`. 

```python
from fastapi import FastAPI, Request

app = FastAPI(title="This is the FastAPI object title")


@app.get("/")
async def root(request: Request):
    return {"app_title": request.app.title}


if __name__ == "__main__":
    import uvicorn

    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Going to the endpoint `localhost:8000` yields the response `{"app_title":"This is the FastAPI object title"}`

**Comment by selobu** [comment]
Thank's a lot

**Comment by tiangolo** [comment]
Thanks for the help here @JarroVGIT ! 👏  🙇 

Thanks for reporting back and closing the issue @selobu  👍
